Cake.Recipe/Content/addins.cake

```diff
@@ -6,7 +6,6 @@
 #addin nuget:?package=Cake.Coveralls&version=0.10.1
 #addin nuget:?package=Cake.Email&version=0.9.1&loaddependencies=true // loading dependencies is important to ensure Cake.Email.Common is loaded as well
 #addin nuget:?package=Cake.Figlet&version=1.3.1
-#addin nuget:?package=Cake.Git&version=0.21.0
```

Comment:
@pascalberger just curious, why did this get removed?  Seemed unrelated to this change.

Reply:
See comment in description: 

>Since Cake.Git already comes with Cake.Issues.Recipe it needs to be removed from Cake.Recipe.

Reply:
Apologies, completely missed that!